README.md

```diff
@@ -168,3 +143,31 @@ A Docker image of PostgreSQL is provided in the `docker-compose.yml` file, you c
 ```bash
 docker-compose up postgres
 ```
+
+## Remaining Tasks
+
+### Short/Mid Term Goals:
+
+- **Enhance SQL Query Performance**: Continuously improve the performance of SQL queries.
+- **Refine Internal Server Error Screens**: Improve the user experience by enhancing error screens.
+- **Implement Preview in the Search Bar**: Enable a preview feature in the Search Bar.
+- **Expand Search Bar Functionality**: Incorporate Events, Messages, Classes, and Contracts into the Search Bar.
+- **Preserve Leading Zeros in Hash Storage**: Ensure that leading zeros are not removed when storing hashes in the database, or adapt the Search Bar to accommodate missing leading zeros.
```

Comment:
IMO this is just a matter of fixing the search as opposed (so it can be included in the point above) to keeping the zeros